xml/System.IO/StreamReader.xml

```diff
@@ -332,7 +332,7 @@
 ## Remarks  
  This constructor initializes the encoding to <xref:System.Text.UTF8Encoding>, the <xref:System.IO.StreamReader.BaseStream%2A> property using the `stream` parameter, and the internal buffer size to 1024 bytes.  
   
- The `detectEncodingFromByteOrderMarks` parameter detects the encoding by looking at the first four bytes of the stream. It automatically recognizes UTF-8, little-endian Unicode, big-endian Unicode, little-endian UTF-32, and big-endian UTF-32 text if the file starts with the appropriate byte order marks. See the <xref:System.Text.Encoding.GetPreamble%2A?displayProperty=nameWithType> method for more information.  
+ The `detectEncodingFromByteOrderMarks` parameter detects the encoding by looking at the first four bytes of the stream. It automatically recognizes UTF-8, little-endian UTF-16, big-endian UTF-16, little-endian UTF-32, and big-endian UTF-32 text if the file starts with the appropriate byte order marks. See the <xref:System.Text.Encoding.GetPreamble%2A?displayProperty=nameWithType> method for more information.  
```

Comment:
```suggestion
 The `detectEncodingFromByteOrderMarks` parameter detects the encoding by looking at the first four bytes of the stream. It automatically recognizes UTF-8, little-endian Unicode (UTF-16), big-endian Unicode (UTF-16), little-endian UTF-32, and big-endian UTF-32 text if the file starts with the appropriate byte order marks. See the <xref:System.Text.Encoding.GetPreamble%2A?displayProperty=nameWithType> method for more information.  
```

Reply:
Personally, I don't like it, since it's incorrect. [According to Wikipedia](https://en.wikipedia.org/wiki/UTF-16#History), the UCS-2 encoding used to be called "Unicode", but that's not even the same encoding as UTF-16.

I get that some places will have to keep using "Unicode" for UTF-16 (like `Encoding.Unicode`), which makes this a bit confusing. But I think a small amount of confusion and inconsistency within .Net is worth it to be consistent with the Unicode standard.

Also, sorry for the huge delay in responding.